test/kubemark/start-kubemark.sh

```diff
@@ -123,93 +124,113 @@ function delete-kubemark-image {
   delete-image "${KUBEMARK_IMAGE_REGISTRY}/kubemark:${KUBEMARK_IMAGE_TAG}"
 }
 
+RESOURCE_SERVER_KUBECONFIG=""
+RP_NUM=""
 # Generate secret and configMap for the hollow-node pods to work, prepare
 # manifests of the hollow-node and heapster replication controllers from
 # templates, and finally create these resources through kubectl.
 function create-kube-hollow-node-resources {
-  # Create kubemark namespace.
-  "${KUBECTL}" create -f "${RESOURCE_DIRECTORY}/kubemark-ns.json"
+  echo "DBG: RP number: ${RP_NUM}"
+  local -r current_rp_kubeconfig=${RP_KUBECONFIG}-${RP_NUM}
+  echo "DBG: PR kubeconfig: ${current_rp_kubeconfig}"
 
-  # Create configmap for configuring hollow- kubelet, proxy and npd.
-  "${KUBECTL}" create configmap "node-configmap" --namespace="kubemark" \
-    --from-literal=content.type="${TEST_CLUSTER_API_CONTENT_TYPE}" \
-    --from-file=kernel.monitor="${RESOURCE_DIRECTORY}/kernel-monitor.json" \
-    --from-literal=resource.server.kubeconfig="${RESOURCE_SERVER_KUBECONFIG:-}" \
-    --from-literal=tenant.server.kubeconfigs="${TENANT_SERVER_KUBECONFIGS:-}"
+  # Create kubemark namespace.
+  if [[ ${RP_NUM} == 1 ]]; then
+    "${KUBECTL}" create -f "${RESOURCE_DIRECTORY}/kubemark-ns.json"
+  fi
```

Comment:
Only when RP_NUM=1 need kubemark namespace? does RP_NUM=2 or more need kubemark namespace?

Reply:
kubemark namespace is only needed to create once at the admin cluster.